sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -18,8 +18,7 @@ param (
 )
 
 ###### setup ######
-Install-Module -Name Az -RequiredVersion 4.8.0 -Force -AllowClobber
-Install-Module -Name Az.DeviceProvisioningServices -Force
+Install-Module -Name Az -RequiredVersion 4.8.0 -Force -AllowClobber -SkipPublisherCheck
```

Comment:
From previous conversation we discussed that `Az.IotHub` needs the latest `Az.Accounts`. Why not install `Az.Accounts` with 2.2.6 as the required minimum version and `Az.IotHub` with an appropriate minimum version?

Reply:
that's what I would ideally like to do. As you can see [here](https://github.com/Azure/azure-sdk-for-c/pull/1625/commits/99b768f978c1284d82bb25e85272f28658c4008b) that's been having its issues. Once I work out the kinks thats the long term solution. Until I figure out why it's not working this opens the gate for the release tomorrow.

Reply:
It's ok to mark this discussion resolved. You tried to get it to do the right thing and it didn't and we need to investigate further. Since this unblocks shipping I'm ok with this change for now and we have an issue to track the resolution: https://github.com/Azure/azure-sdk-for-c/issues/1635 

My concern is that future tests might get broken by this in a way that's hard to determine. This doesn't negatively impact live testing right now.

Reply:
marking as resolved